compiler/luci-interpreter/src/kernels/Pow.test.cpp

```diff
@@ -87,6 +87,16 @@ TEST(PowTest, IntPow)
 }
 
 TEST(PowTest, Input_Output_Type_NEG)
+{
+  Tensor input1_tensor = makeInputTensor<DataType::FLOAT32>({1}, {1.0f});
+  Tensor input2_tensor = makeInputTensor<DataType::FLOAT32>({1}, {1.0f});
+  Tensor output_tensor = makeOutputTensor(DataType::BOOL);
+
+  Pow kernel(&input1_tensor, &input2_tensor, &output_tensor);
+  EXPECT_ANY_THROW(kernel.configure());
+}
+
+TEST(PowTest, Input_Type_Mismatch_NEG)
```

Comment:
(optional) as there are two changs, rename existing `Input_Output_Type_NEG` to `Input_Type_Mismatch_NEG` and add new `Input_Output_Type_NEG` and `Invalid_Input_Type_NEG`, it would be better to split PRs. 
But as this changes is not that complicated I think we can go as-is unless other reviewers want that.

Reply:
I'll be aware of that. Thank you for pointing it out :)